.github/workflows/static-analysis.yml

```diff
@@ -12,6 +12,8 @@ on:
       - "v*.*"
       - "master"
       - "feature/*"
+    tags:
+      - "*"
```

Comment:
The analysis will be performed twice on the same commit: 1) for the push on master, 2) for the tag. I presume it's so that the results are linked correctly at tag level.

Reply:
That is correct.